transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/MessagingClient.kt

```diff
@@ -232,4 +232,20 @@ interface MessagingClient {
      */
     @Throws(IllegalStateException::class)
     fun logoutFromAuthenticatedSession()
+
+    /**
+     * Permanently clears the existing conversation history with an agent from all devices that share the same session.
+     * This action is allowed in both [State.Configured] and [State.ReadOnly].
+     *
+     * After successful clearance, [Event.ConversationCleared] will be dispatched to allow
```

Comment:
is it `Event.Clear` as in the design or `EventConversationCleared`

Reply:
Good point. I am going to update the design. Event.ConversationCleared sounds more correct comparing to Event.Clear which is to ambiguous.